Beskrivelse af hvordan context er modeled anderledes i CSGCN-ADJ (#12)
thesis20/thesis_paper: sections/csgcn/csgcn-chapter.tex

This section describes the second proposed model, CSGCN with Context in Adjacency Matrix (CSGCN-ADJ).

\subsection{Model intuition}\label{subsec:csgcn_adj_intuition}
With the second proposed model, context is included in the convolution by adding context nodes to the graph, which will be used to represent interactions between users and items in a given context as a tuple $(u,i,C')$, implying that a user $u$ has interacted with item $i$ in context $C'$.
The adjacency matrix will now include interactions, what context the users have interacted in, and the context in which items have been interacted with.

> This approach is unable captures which specific context the different interactions occured in.

Reviewer: ```suggestion
This approach is unable to capture which specific contexts the different interactions occured in.
```

Reviewer: skal vi skrive en sætning om at det enten er godt leler skidt?

Reviewer: nvm det står nedeunder

> This approach is unable to capture which specific contexts the different interactions occured in.

Reviewer 2: ```suggestion
This approach is unable to capture which specific contexts the different interactions occurred in.
```

It only captures whether or not the user or item has had an interaction in the given context.

> This differs from \textit{CSGCN-IS} since we now model context in a similar fashion to side-information, where each context value has a single vector representation, instead of having a vector representation for each context value and item pair.

Reviewer: ```suggestion
This differs from \textit{CSGCN-IS} since we now model context in a similar fashion to side-information, where each context value has a single vector representation instead of having a vector representation for each context value and item pair.
```
